os/arch/arm/include/s5j/dma.h

```diff
@@ -21,7 +21,7 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
-#define s5j_dmasetup(a, b) if ((b)->setup) { (b)->setup(a, b) };
+#define s5j_dmasetup(a, b) if ((b)->setup) { (b)->setup(a, b); };
```

Comment:
convert this to do while as below:
#define s5j_dmasetup(a, b) 
do {      \
if ((b)->setup)   \
 (b)->setup(a, b); \ 
} while (0)

Reply:
#define s5j_dmasetup(a, b) \
do {  \
if ((b)->setup)  \
(b)->setup(a, b); \
} while (0)

Reply:
Done